### PR TITLE
update readme and created manifesto

### PR DESCRIPTION
**Pull request by Eduen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,25 +3,86 @@
...
+
+## Manifesto
+
+Manifesto brings together our creed and opinions of the whole project direction. See [MANIFESTO](https://github.com/mrzahrada/mercore/blob/master/MANIFESTO.md)
 
-    cmake . & make & make install
+## Contributing
 
-## How to use Mercore
+The instructions for cooperation will be detailed soon.
```
Comment (snurkabill) [inline README.md]: * "How to coopearte" could be better

File: README.md [hunk 2]
```diff
@@ -3,25 +3,86 @@
...
+`cmake .`
+
+Finish compiling with
+
+`make`
+
+
+## Requirements
+
+To run mercore properly, you will need other essential structures:
```
Comment (snurkabill) [inline README.md]: without "will"

File: README.md [hunk 3]
```diff
@@ -3,25 +3,86 @@
...
+
+* Boost Libraries v. 1.57.0
+* [ØMQ Libraries](http://github.com/zeromq/libzmq) v. 4.0.5
+* [ØMQ C++ Driver](http://github.com/zeromq/cppzmq) v. 4.1 
+* [docopt](http://github.com/docopt/docopt) v. 0.6.1 
+* g++ v. 4.8 *or* clang++ v. 3.7
+* [MongoDB](http://github.com/mongodb/mongo) v. 2.4.12 
+* [Mongo C++ Driver](http://github.com/mongodb/mongo-cxx-driver) v. 1.0.0 
+
+You can prevent hours of frustration if you install all these structures in recommended versions. The first version of mercore is tested on Debian and Ubuntu.
```
Comment (snurkabill) [inline README.md]: "It is strongly recommended use versions above", ... something like "we have no responsibilities if you use different ones"

File: README.md [hunk 4]
```diff
@@ -3,25 +3,86 @@
 [![Join the chat at https://gitter.im/mrzahrada/mercore](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mrzahrada/mercore?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/mrzahrada/mercore.svg?branch=master)](https://travis-ci.org/mrzahrada/mercore)
 
-## Requirements
+## About
+
+Mercore is brought to you by Openmerc community. Openmerc helps you to get done with your activities in trading. You are about to explore all-in-one trading solution with unlimited possibilities. Mercore is the one and only instrument that serves the Openmerc community.
```
Comment (snurkabill) [inline README.md]: "Mercore is the one and only instrument that serves the Openmerc community." it is very strong claim, are you sure about it?

File: README.md [hunk 5]
```diff
@@ -3,25 +3,86 @@
...
+
+Mercore is brought to you by Openmerc community. Openmerc helps you to get done with your activities in trading. You are about to explore all-in-one trading solution with unlimited possibilities. Mercore is the one and only instrument that serves the Openmerc community. 
 
-  * boost libraries 
-  * clang++ or g++  
-  
-Tested on Linux (Debian, Ubuntu).  
+## Issues and Fixes
+
+At the moment, we are not experiencing any bugs and issues. Help us to dig them out.
```
Comment (snurkabill) [inline README.md]: usually, there are words like "hunt them out" or "find them out" 

I've never seen dig + bug in one sentence before.

File: README.md (hunk 2 above)
Comment (mrzahrada) [inline README.md]: mercore is renamed to openmerc and there should be ```cd openmerc```

Comment (Eduen) [inline README.md]: thanks for feedback, I'll correct it asap

File: README.md (hunk 5 above)
Comment (Eduen) [inline README.md]: Details make perfection, thanks.

**Pull request merged**